src/components/forms/SubmitTemplate/server-action.ts

```diff
@@ -30,6 +35,8 @@ export async function submitTemplate(formData: FormData) {
 
     const templateEntity = await saveTemplate(validatedTemplate);
 
+    await deleteSession(sessionId);
```

Comment:
Can we delete and send the email in parallel, especially since the deletion is not critical due to the TTL.

Reply:
good idea!!

Reply:
Updated,